src/psij/job_executor_config.py

```diff
@@ -5,16 +5,22 @@
 class JobExecutorConfig(object):
     """An abstract configuration class for :class:`~psij.JobExecutor` instances."""
 
-    def __init__(self, launcher_log_file: Optional[Path] = None):
+    DEFAULT: 'JobExecutorConfig' = None  # type: ignore
+    DEFAULT_WORK_DIRECTORY = Path.home() / '.psij' / 'work'
```

Comment:
The documentation in the batch scheduler executor misses the `work` part I think?

Reply:
Thanks.

Reply:
Fixed